src/createContext.ts

```diff
@@ -51,44 +53,50 @@ export const defineSymbol = (context: Context, name: string, value: Value) => {
   })
 }
 
+export const defineBuiltin = (context: Context, name: string, f: Function) => {
+  let wrapped = (...args: any) => f(...args)
+  wrapped.toString = () => {
+    const params = getParameterNames(f).join(', ')
+    return `function ${f.name}(${params}) {\n\t[implementation hidden]\n}`
```

Comment:
We should use `name` instead of `f.name`. That's what the students will use. For example, `stringify(math_sin)` will now display that the name of the function is `sin` instead of `math_sin`, which is confusing.